### ISSUE TITLE
Non-visual navigation of spreadsheets  Enhancing accessibility of Microsoft Excel™

### ISSUE THREAD

**Issue by Wotipati** [Wotipati/Blind_Accessibility_Papers#315]
### Links
- Paper: https://link.springer.com/article/10.1007/s10209-012-0272-1

<img width="676" alt="ScreenShot 2020-05-25 19 09 34" src="https://user-images.githubusercontent.com/26996041/82803348-53c95d80-9ebb-11ea-9ebf-19b2f63b9165.png"><img width="648" alt="ScreenShot 2020-05-25 19 09 43" src="https://user-images.githubusercontent.com/26996041/82803352-54fa8a80-9ebb-11ea-98a1-96dbca732a48.png">

### Abstract
The problem of non-visual navigation of the information in Excel™ spreadsheet is that using current technologies, no overview is available for the user about different components found in the spreadsheet. Several attributes associated with spreadsheets make them not easy to navigate by individuals who are blind. The large amount of information stored in the spreadsheet, the multi-dimensional nature of the contents, and the several features it includes cannot be readily linearized by a screen reader or Braille display (e.g., charts and tables). A user-centered design paradigm is followed to build an accessible system for non-visual navigation of Microsoft Excel™ spreadsheet. The proposed system provides the user with a hierarchical overview of the navigated components in an Excel™ spreadsheet. The system is multi-modal, and it provides the user with different navigation and reading modes for the non-visual navigation of a spreadsheet. This will help the users in adapting the non-visual navigation according to the task the user needs to accomplish.
- Excelをアクセシブルにするシステムを提案
